src/containers/BooksList.js

```diff
@@ -17,6 +17,7 @@ const BooksList = () => {
           <th>Book ID</th>
           <th>Title</th>
           <th>Category</th>
+          <th>Action</th>
         </tr>
       </thead>
       <tbody>
```

Comment:
**REQUIRED** 
* It implements handleBookRemove ([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)).
* It connects REMOVE_BOOK action.
* It passes handleRemoveBook to Book component [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)